File: Server/BaseRepository/Frutas/Models/Carrinho.cs
using Base.Repository.Models;
using System.Collections.Generic;

namespace Frutas.Models
{
    public class Carrinho: Entity
    {
        public List<ItemCarrinho> Itens { get; set; }
        public double TotalPedido { get; set; }
        public bool Finalizado { get; set; } = false;
    }
}

File: Client/src/domain/fruta/Carrinho.js
import Item from './ItemCarrinho';
export default class Carrinho {

    constructor(TotalPedido=0.00, Finalizado=false, Itens = []) {

        this.totalPedido = TotalPedido;
        this.finalizado = Finalizado;
        this.Itens = Itens;
    }
}